tests/robot-tests/tests/admin_and_public/bau/archive_publication.robot

```diff
@@ -0,0 +1,293 @@
+*** Settings ***
+Library             ../../libs/admin_api.py
+Resource            ../../libs/admin-common.robot
+Resource            ../../libs/public-common.robot
+
+Suite Setup         user signs in as bau1
+Suite Teardown      user closes the browser
+Test Setup          fail test fast if required
+
+Force Tags          Admin    Local    Dev    AltersData
+
+*** Variables ***
+${PUB_NAME_ARCHIVE}=            UI tests - archived publication %{RUN_IDENTIFIER}
```

Comment:
Would rename this & any other instance of `PUB` to either `publication` or `public` as it could lead to confusion about whether this means 'public' or 'publication'